docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -331,7 +335,7 @@ usage, before the computational VMs are deployed.
    If you refresh the
    [VM instances page](https://console.cloud.google.com/compute/instances) you
    will see an `a2-highgpu-1g` machine that has been auto-scaled up to run this
-   job. It will have a name like `hcls01-gpu-ghpc-0`.
+   job. It will have a name like `hclsclusterv6-gpu-ghpc-0`.
```

Comment:
Did you test this? I thought cluster name was limited to 10 char. I think you are showing 13.

Reply:
Moved this back to previous one. Since at this point we recommend Slurm-gcp v5 so we shouldn't update the doc.